microovn/Makefile

```diff
@@ -0,0 +1,11 @@
+DIRNAME := $(shell basename $(CURDIR))
+CHARMFILE := microovn_ubuntu-22.04-amd64.charm 
+
+build: $(CHARMFILE)
+
+$(CHARMFILE): src/charm.py charmcraft.yaml
+	charmcraft pack
```

Comment:
nit (for both charms): Adding `-v` for verbose output would be nice. It'll make debugging builds, especially in CI/launchpad easier.